### PR TITLE
Backups api v2

### PR DESCRIPTION
**Pull request by xiwenc**
Earlier this week the new Backups API V2 was released. Now we are updating the public documentation to encourage users to use V2.

@MarkvanMents could you review this?

### CODE REVIEW

File: content/apidocs-mxsdk/apidocs/backups-api-v1.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
+---
+title: "Backups API"
+parent: "deploy-api"
+description: "An API to allow the triggering of backups creation, restore, download and to get information about existing snapshots."
+---
+
+## 1 Introduction
+
+The Backups API allows you to manage data snapshots of applications hosted in the Mendix Cloud. Data snapshots consist of a Postgresql database dump and file objects referenced from the database. You can create new snapshots, and restore or download them. Uploading snapshots is currently only supported via the [Developer Portal](/developerportal/operate/backups).
+
```
Comment (MarkvanMents) [inline content/apidocs-mxsdk/apidocs/backups-api-v1.md]: @xiwenc 
Should we add something here to say that this is v1 and that there is a later version?

Comment (xiwenc) [reply content/apidocs-mxsdk/apidocs/backups-api-v1.md]: yes that's a great idea :) Could you add it?

Comment (MarkvanMents) [reply content/apidocs-mxsdk/apidocs/backups-api-v1.md]: @xiwenc - can you let me know if this is OK - then I can publish it.
Thanks for writing the documentation.

Comment (xiwenc) [reply content/apidocs-mxsdk/apidocs/backups-api-v1.md]: yep looks good

**Comment by xiwenc** [comment]
Thanks a lot for the review and corrections!

**Pull request merged**